connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerTask.java

```diff
@@ -184,8 +184,7 @@ private void doRun() throws InterruptedException {
 
             execute();
         } catch (Throwable t) {
-            log.error("{} Task threw an uncaught and unrecoverable exception", this, t);
-            log.error("{} Task is being killed and will not recover until manually restarted", this);
+            log.error("{} Task threw an uncaught and unrecoverable exception, task is being killed and will not recover until manually restarted", this, t);
```

Comment:
I'd suggest keeping the format slightly closer to what we had before. Specifically:

`Task threw an uncaught and unrecoverable exception. Task is being killed and will not recover until manually restarted`

Reply:
I also like @kkonstantine's suggested format.